packages/shared/src/store/useChannelPosts.ts

```diff
@@ -1,37 +1,47 @@
 import { useInfiniteQuery } from '@tanstack/react-query';
-import { useEffect } from 'react';
+import { useEffect, useMemo } from 'react';
 
 import * as db from '../db';
 import { createDevLogger } from '../debug';
 import * as sync from './sync';
 import { useKeyFromQueryDeps } from './useKeyFromQueryDeps';
 
-const postsLogger = createDevLogger('useChannelPosts', false);
+const postsLogger = createDevLogger('useChannelPosts', true);
```

Comment:
Do we need this to be true?
```suggestion
const postsLogger = createDevLogger('useChannelPosts', false);
```